background/get-userscripts.js

```diff
@@ -172,9 +172,13 @@ scratchAddons.localEvents.addEventListener("themesUpdated", () => {
       if (tab.url || (!tab.url && typeof browser !== "undefined")) {
         chrome.tabs.sendMessage(tab.id, "getInitialUrl", { frameId: 0 }, async (res) => {
           if (res) {
+            const csInfo = await getContentScriptInfo(res);
             chrome.tabs.sendMessage(
               tab.id,
-              { themesUpdated: (await getContentScriptInfo(res)).themes },
+              {
+                themesUpdated: csInfo.themes,
+                addonsWithUserstyles: csInfo.addonsWithUserstyles,
```

Comment:
This is not a good idea, we never send two messages at once (they all have one key)